benchmark/blankenbach/base_case1b.prm

```diff
@@ -0,0 +1,19 @@
+include base_case1a.prm
+
+set Additional shared libraries            = ./plugin/libblankenbach.so
+
+subsection Initial conditions
+  set Model name = ascii data
+  subsection Ascii data model
+    set Data directory = 
+    set Data file name = initial_temperature_case1b.txt
+  end
+end
+
+
+subsection Material model
+  subsection Nondimensional model
```

Comment:
maybe just to make clear that this is the model that's actually used, repeat the `set Model = nondimensional` line here?